src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/softmax_kernel.hpp

```diff
@@ -0,0 +1,576 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Why it was extracted in separate header?

Reply:
Softmax is used in 2 places: one is in [second token](https://github.com/openvinotoolkit/openvino/blob/72c52f2b0adef6ee97d15e40e7dbcc728a0730b6/src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/mha_single_token.cpp#L175-L184), the other is in [fisrt token](https://github.com/openvinotoolkit/openvino/blob/72c52f2b0adef6ee97d15e40e7dbcc728a0730b6/src/plugins/intel_cpu/src/nodes/scaled_attn.cpp#L280).
Moving it to header can make it inline in second token.